kafka-ui-react-app/src/components/common/Dashboard/__tests__/__snapshots__/Indicator.spec.tsx.snap

```diff
@@ -1,27 +1,29 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Indicator matches the snapshot 1`] = `
-<Indicator
+<Styled(Indicator)
   label="label"
   title="title"
 >
-  <div
-    className="level-item"
+  <Indicator
+    className="sc-bdfBQB eGAtcZ"
+    label="label"
+    title="title"
   >
     <div
-      title="title"
+      className="sc-bdfBQB eGAtcZ"
```

Comment:
Hashed classnames in snapshots may cause unreadable error in tests after styles changes. Check this, please https://github.com/styled-components/jest-styled-components
Not critical change, just FYI